### PR TITLE
Ar/add logging for server restart and usage of sudo

### PR DESCRIPTION
**Pull request by angyangie**
Adding more clear instructions: https://github.com/livepeer/devops/issues/12

### CODE REVIEW

**Comment by angyangie** [comment]
After this PR, will re-clone livepeer-wowza to get rid of all the unnecessary merge commits from master up there ^

File: installation_script.go [hunk 1]
```diff
@@ -71,6 +71,7 @@ func main() {
 	// Download latest JAR file to disk
 	err = downloadLatestJarFile(*channel, *wowzaDir)
 	if err != nil {
+		fmt.Print("*** You must run this installation script using as an admin using `sudo`\n")
```
Comment (iameli) [inline installation_script.go]: Possible to do any kind of checking for the "permission denied" error specifically so we can tell the difference between this and stuff like a "no internet" error? Here's what I get on the current version:
```
./install_livepeer_wowza
Running on a linux operating system
Livepeer API key in Server.XML: [redacted]
✔ No
*** The following API key already provided: '[redacted]'. Would you like to enter a new Livepeer API key? Your answer: No
panic: Download latest .jar file error: Os.Create error: open /usr/local/WowzaStreamingEngine/lib/LivepeerWowza.jar: permission denied

goroutine 1 [running]:
main.main()
	/go/src/github.com/livepeer/livepeer-wowza/installation_script.go:74 +0xb33
```

Comment (angyangie) [reply installation_script.go]: Yesss! Pushing change now

**Comment by iameli** [comment]
LGTM!

**Pull request merged**